src/app.ts

```diff
@@ -138,6 +138,17 @@ async function app() {
     );
     igExporter.export(outDir);
     logger.info('Assembled Implementation Guide sources; ready for IG Publisher.');
+    if (
+      !isIgPubContext &&
+      !isLegacyIgPubContext &&
```

Comment:
I actually think it was a mistake before for us not to include these scripts in publisher contexts.  It confused people -- and since most people actually _started_ with SUSHI they still needed the scripts.

Anyway, I think it doesn't matter what context we're in.  If we don't see the scripts, let's log that info message no matter what.